frontend/src/components/elements/ArrowVegaLiteChart/CustomTheme.tsx

```diff
@@ -202,7 +110,7 @@ export function applyStreamlitTheme(config: any, theme: Theme): any {
         : { color: "#83C9FF" }),
```

Comment:
nit: Should this be also somehow added to the utils?

Reply:
Yes! I'll add that. Good catch thanks